rtv/page.py

```diff
@@ -115,9 +166,19 @@ def __init__(self, stdscr, reddit, content, **kwargs):
         self._content_window = None
         self._subwindows = None
 
+    @BaseController.register('q')
+    def exit(self):
+        sys.exit()
+
+    @BaseController.register('?')
+    def help(self):
+        show_help(self.stdscr)
+
+    @BaseController.register(curses.KEY_UP, 'k')
     def move_cursor_up(self):
         self._move_cursor(-1)
```

Comment:
Looks like these are missing the ``self.clear_input_queue()`` that they used to have.

Basically what that does is prevent holding down the arrow keys or using the scroll wheel from building up a bunch of backlogged commands and lagging the screen once the key is released.